src/latency.c

```diff
@@ -628,6 +768,20 @@ void latencyCommand(client *c) {
                 resets += latencyResetEvent(c->argv[j]->ptr);
             addReplyLongLong(c,resets);
         }
+    } else if (!strcasecmp(c->argv[1]->ptr,"histogram") && c->argc >= 2) {
+        /* LATENCY HISTOGRAM*/
+        if (c->argc == 2) {
+            latencyAllCommandsFillCDFResp(c);
+        } else {
+            latencyCommandFillCDFResp(c);
+        }
+    } else if (!strcasecmp(c->argv[1]->ptr,"histogram-report") && c->argc >= 2) {
+        /* LATENCY HISTOGRAM-REPORT*/
+        if (c->argc == 2) {
+            latencyAllCommandsFillCDF(c);
```

Comment:
maybe these should have a "Human" suffix to make it clearer what's the difference.